.ci/podSpecs/integration-test.yml

```diff
@@ -47,8 +47,8 @@ spec:
       tty: true
       resources:
         limits:
-          cpu: 8
-          memory: 8Gi
+          cpu: 16
+          memory: 24Gi
         requests:
-          cpu: 8
-          memory: 8Gi
+          cpu: 10
```

Comment:
I think it's find to start this at 8 cpu or move it to 12 cpu, and limit to 16. 10 seems like a strange number of cpu's 😄

Reply:
All these numbers are arbitrary, to be honest. All I can see is if I lower them it flakes more often. Let's talk about it over Zoom, because it's clearly a resource/parallelism problem, but I can't quite tell what's the right solution.